### PR TITLE
DX: Add return types

### PR DESCRIPTION
**Pull request by kubawerlos**
Follow up to https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5934

### CODE REVIEW

File: src/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixer.php [hunk 1]
```diff
@@ -157,7 +157,7 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                         if (!isset($default[$from])) {
                             throw new InvalidOptionsException(sprintf(
                                 'Unknown key "%s", expected any of "%s".',
-                                \is_object($from) ? \get_class($from) : \gettype($from).(\is_resource($from) ? '' : '#'.$from),
```
Comment (keradus) [inline src/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixer.php:160]: cannot form be object/resource?

Comment (kubawerlos) [reply src/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixer.php:160]: No, it cannot, see line 152, it's a key of an array.

**Review by GrahamCampbell — APPROVED** [review]

**Comment by GrahamCampbell** [comment]
Merge conflicts. :(

**Review by GrahamCampbell — APPROVED** [review]
🚢

**Comment by GrahamCampbell** [comment]
Might be good to merge this one now @SpacePossum before more conflicts. :)

**Comment by GrahamCampbell** [comment]
:shipit:

**Comment by GrahamCampbell** [comment]
More merge conflicts. 😭

**Review by keradus — APPROVED** [review]

**Comment by SpacePossum** [comment]
Thank you @kubawerlos.

**Pull request merged**